src/tests/api/HappyPath.test.ts

```diff
@@ -0,0 +1,244 @@
+	])("Successful Request Tests - $yotiMockId", async ({ buildingNumber, buildingName, subBuildingName }: { buildingNumber: string; buildingName: string; subBuildingName: string }) => {
+		const newf2fStubPayload = structuredClone(f2fStubPayload);
+		newf2fStubPayload.shared_claims.address[0].buildingNumber = buildingNumber;
+		newf2fStubPayload.shared_claims.address[0].buildingName = buildingName;
+		newf2fStubPayload.shared_claims.address[0].subBuildingName = subBuildingName;
+		const { sessionId } = await startStubServiceAndReturnSessionId(newf2fStubPayload);
+
+		const response = await postDocumentSelection(dataUkDrivingLicence, sessionId);
+		expect(response.status).toBe(200);
+
+		await getSessionAndVerifyKey(sessionId, constants.DEV_F2F_SESSION_TABLE_NAME, "authSessionState", "F2F_YOTI_SESSION_CREATED");
+	});
+});
+
+
+describe("/authorization endpoint", () => {
+	it.each([
+		{ yotiMockId: "0000", docSelectionData: dataUkDrivingLicence, yotiStartSchema: "F2F_YOTI_START_00_SCHEMA" },
+		{ yotiMockId: "0100", docSelectionData: dataPassport, yotiStartSchema: "F2F_YOTI_START_SCHEMA" },
+		{ yotiMockId: "0200", docSelectionData: dataNonUkPassport, yotiStartSchema: "F2F_YOTI_START_SCHEMA" },
+		{ yotiMockId: "0300", docSelectionData: dataBrp, yotiStartSchema: "F2F_YOTI_START_03_SCHEMA" },
+		{ yotiMockId: "0400", docSelectionData: dataEuDrivingLicence, yotiStartSchema: "F2F_YOTI_START_00_SCHEMA" },
+		{ yotiMockId: "0500", docSelectionData: dataEeaIdCard, yotiStartSchema: "F2F_YOTI_START_05_SCHEMA" },
+	])("Successful Request Tests - $yotiMockId", async ({ yotiMockId, docSelectionData, yotiStartSchema }: { yotiMockId: string; docSelectionData: DocSelectionData; yotiStartSchema: string }) => {
+		f2fStubPayload.yotiMockID = yotiMockId;
+		const { sessionId } = await startStubServiceAndReturnSessionId(f2fStubPayload);
+
+		await postDocumentSelection(docSelectionData, sessionId);
+
+		const authResponse = await authorizationGet(sessionId);
+		expect(authResponse.status).toBe(200);
+
+		await getSessionAndVerifyKey(sessionId, constants.DEV_F2F_SESSION_TABLE_NAME, "authSessionState", "F2F_AUTH_CODE_ISSUED");
+
+		const allTxmaEventBodies = await getTxmaEventsFromTestHarness(sessionId, 5);
+		validateTxMAEventData({ eventName: "F2F_CRI_START", schemaName: "F2F_CRI_START_SCHEMA" }, allTxmaEventBodies);
+		validateTxMAEventData({ eventName: "F2F_YOTI_START", schemaName: yotiStartSchema }, allTxmaEventBodies);
+		validateTxMAEventData({ eventName: "F2F_YOTI_PDF_EMAILED", schemaName: "F2F_YOTI_PDF_EMAILED_SCHEMA" }, allTxmaEventBodies);
+		validateTxMAEventData({ eventName: "F2F_CRI_AUTH_CODE_ISSUED", schemaName: "F2F_CRI_AUTH_CODE_ISSUED_SCHEMA" }, allTxmaEventBodies);
+		validateTxMAEventData({ eventName: "F2F_CRI_END", schemaName: "F2F_CRI_END_SCHEMA" }, allTxmaEventBodies);
+	});
+});
+
+describe("/token endpoint", () => {
+	it.each([
+		{ yotiMockId: "0000", docSelectionData: dataUkDrivingLicence, yotiStartSchema: "F2F_YOTI_START_00_SCHEMA" },
```

Comment:
We don't seem to be using the `yotiStartSchema` anywhere in these tests, is that because it's not needed?

Reply:
Actually same with the `/userinfo endpoint` tests

Reply:
Yeah no additional TxMA events are being triggered as part of these 2 endpoints so no TxMA validation logic has been added. 

Those unused schemas have been removed